packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -16,9 +16,11 @@ const styles: any = stylesImport;
 
 const leftArrow = 'Up';
 const rightArrow = 'Down';
+const closeIcon = 'CalculatorMultiply';
```

Comment:
We use `Cancel` elsewhere.. not sure if that's the same glyph as this. This does fit-in with the UI if `Cancel` doesn't.

Reply:
Cancel is a bit too bold unfortunately, so I think this works better with the existing UI.